geth/node/node.go

```diff
@@ -196,16 +196,20 @@ func activateShhService(stack *node.Node, config *params.NodeConfig, deliverySer
 
 		// enable mail service
 		if whisperConfig.EnableMailServer {
-			password, err := whisperConfig.ReadPasswordFile()
-			if err != nil {
-				return nil, err
+			if whisperConfig.Password == "" {
+				password, err := whisperConfig.ReadPasswordFile()
+				if err != nil {
+					return nil, err
+				}
+
+				whisperConfig.Password = string(password)
```

Comment:
I would avoid changing config here. We could do that in `cmd/wnode-status/config.go`. Otherwise, if the password is used anywhere else, you will need to duplicate this code.

Reply:
As far as i understand we need to have password not only in wnode-status (for mail server) but for clients too. So we cant use `makeNodeConfig()` for this purposes here.

Reply:
As far as I know, a symmetric key will be sent in a Whisper message and that message will be prepared on the status-react side. status-react will just call `symKeyID = shh.generateSymKeyFromPassword()` and use its value in `shh.receiveMessages()` params. We can't hardcode it on the status-go level as it will be coupled too much.

Reply:
Ok, it's for your responsibility - i'll revert my changes.